wiki/C++-Style-Guide.md

```diff
@@ -464,9 +464,9 @@ For example, perhaps your editor has a "feature" that lets you automatically ins
 ```
 But, those two lines of asterisks are eye-grabbing and completely unnecessary on any editor that supports syntax coloring, and more importantly anyone using a different editor would have to manually type an asterisk at the beginning of every comment line and figure out exactly how many asterisks are in those rows at the top and bottom, in order to be consistent with your code.
 
-## SVN commit comments
+## Commit messages
 
-When committing code to SVN, include comments describing everything that is changing.
+When committing code, include comments describing everything that is changing.
 If you are committing multiple changes, it is best to group them into separate commits that each change one thing, rather than committing them all together.
```

Comment:
We can probably just get rid of this. We don't really enforce any requirements on commit descriptions from contributors, since we squash merge so we can write our own descriptions anyway.
Or, move it to "best practices" since that's essentially what it is.